assets/store_descriptions/generate_and_validate.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -xv
+
+# Checks if play store descriptions does not exceed char limits
+# https://support.google.com/googleplay/android-developer/answer/113469?hl=en
+
+error=0
+
+full_description_template="assets/store_descriptions/full-description.txt.template"
+string_base="assets/store_descriptions/"
+resource_base="fastlane/metadata/android/"
+
+for folder in $string_base*
+do
+    [ ! -d "$folder" ] && continue
+    if [ -f "${folder}/strings.lng" ]
+    then
+        source "${folder}/strings.lng"
+        if [[ -z $intro || -z $whatis || -z $rules || -z $supported || -z $bindings || -z $home_automation_solutions || -z $lighting || -z $heating || -z $home_entertainment || -z $security || -z $open_protocols || -z $special_useCases || -z $oss_community || -z $forum || -z $report_issues || -z $translation || -z $foundation || -z $about_foundation || -z $important_note || -z $oh_server || -z $short_description ]]
```

Comment:
I'd recommend the strings-missing-solution I've posted above

Reply:
Where did you post it? Cannot find it.
With my solution no description is created, when one string is missing.

Reply:
I meant this: 
```
source "$resource_base/en-US/strings.lng"
source "$folder/strings.lng"
```

Or in other words: If one string is missing just use the english one. That should be a good compromise and is definitely better then nothing, i.e. showing the english version completely...